runtime/altair/src/lib.rs

```diff
@@ -1810,6 +1819,7 @@ construct_runtime!(
 		// our pallets
 		Fees: pallet_fees::{Pallet, Call, Storage, Config<T>, Event<T>} = 90,
 		Anchor: pallet_anchors::{Pallet, Call, Storage} = 91,
+		Swaps: pallet_swaps::{Pallet, Storage} = 92,
```

Comment:
Maybe CR: IMO we shouldn't re-use indices of removed pallets. In this case, it does not matter. But in general we could receive an XCM pointing to the incorrect pallet index.

Reply:
I didn't know that. Thanks for pointing it out!

Given that these indexes were not used by XCM, maybe we can reuse this as an exception? And fill the gap? If not I revert it

Reply:
Would be interested in @mustermeiszer's thoughts. I think it would be fine to do an exception here because wallets require the latest metadata which will apply this.

Reply:
I am not 100% sure tbh. The fn-signature will not match so the tx would be invalid I guess. In general I think we can re-use indices but given that this is blockchain maybe lets not do until we ran out of indices ^^.

Reply:
Ok, then I move out to a new index